dev/lock.yml

```diff
@@ -3,6 +3,7 @@ dev:
     dev/LICENSE-MPL-2.0.md:
       - typegraph/python/LICENSE.md
       - typegraph/node/LICENSE.md
```

Comment:
the parent package for the workspace can be quite empty (no need for the license or other attributes) if I am not mistaken?

Reply:
yes, `pnpm-workspace.yaml` is the minimum required file.